protocols/grpc/agent.proto

```diff
@@ -170,24 +174,17 @@ message Route {
 	string device = 3;
 }
 
-enum UpdateType {
-	None = 0;
-	AddIP = 1;
-	RemoveIP = 2;
-	Name = 4;
-	MTU = 8;
-}
-
 message AddInterfaceRequest {
-	Interface interface = 1;
+	string interface_id = 1;
+	Interface interface = 2;
 }
 
 message RemoveInterfaceRequest {
-	string name = 1;
+	string interface_id = 1;
```

Comment:
The MAC address should be used here.

Reply:
Sorry, I need to ask,  why `name` can't identify an interface? I maybe missed something you have discussed